CONTRIBUTING.md

```diff
@@ -94,137 +65,217 @@ script before each commit and abort the commit on error. To
+
+### Test with Hot Reload and Debug
 
-### Backend
+Test changes by loading changes locally without having to re-package and re-publish the extension in the marketplace.
 
-#### Overview
+---
 
-The Retrospectives tool uses the [Azure SignalR service](https://azure.microsoft.com/en-us/services/signalr-service/) to add real time support. The backend codebase can be found [here](https://github.com/microsoft/vsts-extension-retrospectives/tree/master/RetrospectiveExtension.Backend).
+#### Hot Reload Prerequisites
+
+1. [Azure DevOps Extension Hot Reload and Debug](https://github.com/microsoft/azure-devops-extension-hot-reload-and-debug)
+
+2. [Visual Studio Code](https://code.visualstudio.com/download)
+
+3. [Firefox](https://www.mozilla.org/en-US/firefox/)
+
+4. [Debugger for Firefox](https://marketplace.visualstudio.com/items?itemName=firefox-devtools.vscode-firefox-debug) VS Code extension
+
+---
+
+1. In the 'RetrospectiveExtension.Frontend' folder, create the 'vss-extension-dev.json' file using the template file `vss-extension-dev.json.template` for reference.
+2. Update the 'webpack.config.js' to enable source maps. Set the devtool property to `inline-source-map`. Also set devServer.https to true and devServer.port to 3000.
+
+    ```js
+    module.exports = {
+      devtool: 'inline-source-map',
+      devServer: {
+        https: true,
+        port: 3000,
+        static: {
+          directory: path.join(__dirname),
+        }
+      },
+    ...
+    ```
+
+3. Set `output.publicPath` to `/dist/` in the webpack.config.json file. This will allow webpack to serve files from `https://localhost:3000/dist`.
+
+    ```js
+    module.exports = {
+        output: {
+          publicPath: "/dist/"
+          // ...
+        }
+        // ..
+    };
+    ```
+
+4. In the root of the project, create a folder named `.vscode`. In there, create a file named `launch.json`, which will help to set up a debug configuration for VS Code that launches Firefox with the correct path mappings. Inside of this file, you will add a path mapping with `url` set to `webpack:///` and have the path set to `${workspaceFolder}/RetrospectiveExtension.Frontend/`. Also set the reAttach property on the configuration to true to avoid restarting Firefox every time you debug.
+
+    ```json
+    {
+      "version": "0.2.0",
+      "configurations": [
+        {
+          "name": "Launch Firefox",
+          "type": "firefox",
+          "request": "launch",
+          "url": "https://localhost:3000/",
+          "reAttach": true,
+          "pathMappings": [
+            {
+              "url": "webpack:///",
```

Comment:
@argoldbe to confirm whether we need to add more path mappings to make it work for everyone.

Reply:
Thanks @phongcao - added suggested change above.